packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/TradeActions.tsx

```diff
@@ -59,10 +46,14 @@ export const TradeActions = ({
 
     const isAccountLoading = selectedAccount ? selectedAccount.status === 'loading' : false;
 
+    const { contentWidth } = useResponsiveContext();
+    const isContentAreaLarge = contentWidth ? contentWidth > breakpointThresholds.lg : true;
```

Comment:
This is being done in 2 places.

Id' create a containgin component for that exactly which would wrap what you want to be displayed in that state. (effectively being if-else, or in this case, just <if>)

Reply:
Good point 👍